src/hotspot/cpu/riscv/macroAssembler_riscv.cpp

```diff
@@ -2016,6 +2016,13 @@ void MacroAssembler::andi(Register Rd, Register Rn, int64_t imm, Register tmp) {
   }
 }
 
+// rotate vector register left with shift bits, 32-bit version
+void MacroAssembler::vrol_vwi(VectorRegister vd, uint32_t shift, VectorRegister tmp_vr) {
+  vsrl_vi(tmp_vr, vd, 32 - shift);
```

Comment:
Nit: You could even inline that in `macroAssembler_riscv.hpp` to match the other "vector pseudo instructions".